core/common/src/test/java/alluxio/network/TieredIdentityFactoryTest.java

```diff
@@ -81,6 +82,7 @@ public void fromScript() throws Exception {
 
   @Test
   public void fromScriptClasspath() throws Exception {
+    JdkTestUtils.assumeJdk8();
```

Comment:
can you add a comment for why this is required?

Reply:
done